src/slave/containerizer/mesos/paths.cpp

```diff
@@ -654,6 +654,30 @@ Option<ContainerID> parseCgroupPath(
   return current;
 }
 
+namespace cgroups2 {
+
```

Comment:
Could have additionally added:
```cpp
// Path to the top-level Mesos cgroup. All global resource constraints
// are impose from this cgroup.
std::string root(const std::string& root)
{
  return root;
}
```

But given that it takes the flag called `cgroups_root` and simply just returns its parameter I didn't see much of a point.